.teamcity/_self/lib/customBuildType/E2EBuildType.kt

```diff
@@ -138,6 +138,7 @@ open class E2EBuildType(
 					xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%E2E_WORKERS% --group=$testGroup
 				"""
 				dockerImage = "%docker_image_e2e%"
+				dockerRunParameters = "-u %env.UID% --shm-size=3g"
```

Comment:
This overrides the default parameter of `bashNodeScript` above to assign 3GB of shared memory only for the `Run Tests` step in `E2EBuildType`.